version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "3.12.0-SNAPSHOT"
+version in ThisBuild := "3.13.0-SNAPSHOT"
```

Comment:
Given the changes here, should we do a major revision? This seems like we introduced some changes we know are not compatible. WDYT?